src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -111,12 +110,12 @@ void UpdateHVACInterface(EnergyPlusData &state,
 
     if ((CalledFrom == DataConvergParams::CalledFrom::AirSystemDemandSide) && (OutletNode == 0)) {
         // Air loop has no return path - only check mass flow and then set return inlet node mass flow to sum of demand side inlet nodes
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACMassFlowNotConverged(1) = false;
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACHumRatNotConverged(1) = false;
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACTempNotConverged(1) = false;
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACEnergyNotConverged(1) = false;
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACEnthalpyNotConverged(1) = false;
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACPressureNotConverged(1) = false;
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACMassFlowNotConverged[0] = false;
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACHumRatNotConverged[0] = false;
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACTempNotConverged[0] = false;
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACEnergyNotConverged[0] = false;
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACEnthalpyNotConverged[0] = false;
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACPressureNotConverged[0] = false;
```

Comment:
Changed all the Array1D parentheses function calls into regular array bracket lookup, decrementing the index everywhere.

Reply:
Here is something else that I would suggest and if I thought Regex could do this or I would suggest that, but these kinds of things should be converted to:

```cpp
auto &airLoopConv = state.dataConvergeParams->AirLoopConvergence(AirLoopNum);
airLoopConv. ... 
```

Just makes the code much more compact and easier to read IMO.

Reply:
Added a bunch of these in my next commit.